packages/vm/src/wasm_backend/gas.rs

```diff
@@ -0,0 +1,170 @@
+use crate::backend::{Querier, Storage};
```

Comment:
This whole file is a TODO/placeholder, right? (I know you said metering was still pending upstream)

Reply:
Right, lots of noop and outdated code here